varya/style.css

```diff
@@ -39,52 +38,51 @@ Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
 	--responsive--aligndefault-width: 100%;
 	--responsive--alignwide-width: 100%;
 	--responsive--alignfull-width: 100%;
-	--responsive--aligndefault-width: 100%;
 	--responsive--alignwide-width-multiplier: calc(16 * var(--global--spacing-horizontal));
 	--responsive--alignright-margin: var(--global--spacing-horizontal);
 	--responsive--alignleft-margin: var(--global--spacing-horizontal);
 }
 
-@media only screen and (min-width: 560px) {
+@media only screen and (min-width: 482px) {
 	:root {
-		--responsive--aligndefault-width: calc(560px - var(--responsive--spacing-horizontal));
-		--responsive--alignwide-width: calc(560px - var(--responsive--spacing-horizontal));
+		--responsive--aligndefault-width: calc(482px - var(--responsive--spacing-horizontal));
+		--responsive--alignwide-width: calc(482px - var(--responsive--spacing-horizontal));
 		--responsive--alignright-margin: calc( 0.5 * (100vw - var(--responsive--aligndefault-width)));
 		--responsive--alignleft-margin: calc( 0.5 * (100vw - var(--responsive--aligndefault-width)));
 	}
 }
 
-@media only screen and (min-width: 640px) {
+@media only screen and (min-width: 592px) {
 	:root {
```

Comment:
Looks like the responsive logic won’t work at all in ie11 even with this plugin :-/ 

The `--responsive--align****-width` variables are updated on `:root` based on the viewport size. This greatly reduces the amount of CSS needed to achieve responsiveness, but it prevents the fallbacks from appearing where they need to be for the PostCSS plugin. 

To get this to work, we may need to refactor and introduce an `%extend` to group all the responsive rules together so the plugin can process the resulting values properly. This comes with the downside of adding a fairly big chunk of new CSS responsive with their accompanying fallbacks. We could also refactor this with a normal `@mixin`, but that would produce even more new CSS lines in an already pretty large CSS file. 

Cc: @kjellr as this might be a blocker to providing full ie11 support if the suggested solutions don’t work.

Reply:
Ok. I think we should de-prioritize this part of the issue for now. We can come back to this closer to launch. 👍

Reply:
Sounds good.